packages/calcite-components/src/components/input-message/input-message.e2e.ts

```diff
@@ -142,4 +143,25 @@ describe("calcite-input-message", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      themed(html`<calcite-input-message>Message</calcite-input-message>`, {
+        "--calcite-input-message-icon-color": {
+          targetProp: "--calcite-input-message-icon-color",
```

Comment:
@Elijbet I just noticed that these target properties are incorrect. They should be targeting `--calcite-icon-color`, and `marginBlockStart`. Can you open a PR to fix these?

Reply:
[refactor(input-message): status themed tests](https://github.com/Esri/calcite-design-system/pull/12701#top)#12701